runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsCounterPlugin.java

```diff
@@ -145,6 +145,8 @@ public class DpsCounterPlugin extends Plugin
 	@Getter(AccessLevel.PACKAGE)
 	private final DpsMember total = new DpsMember("Total");
 
+	private boolean bossDied;
```

Comment:
Reset this on `shutDown()`

Reply:
Fixed in 5b0b2dccde0a52144f3a5d67eb5797c8bb938011